DependencyInjection/Compiler/ThemeCompilerPass.php

```diff
@@ -28,8 +28,12 @@ public function process(ContainerBuilder $container)
                 ->replaceArgument(2, null);
         }
 
-        $twigFilesystemLoaderDefinition = $container->getDefinition('twig.loader.filesystem');
+        $twigFilesystemLoaderDefinition = $container->findDefinition('twig.loader.filesystem');
         $twigFilesystemLoaderDefinition->setClass($container->getParameter('liip_theme.filesystem_loader.class'));
+        $twigFilesystemLoaderDefinition->setArguments(array(
+            $container->getDefinition('liip_theme.templating_locator'),
+            $container->getDefinition('templating.filename_parser')
```

Comment:
That line seems to be the root cause of the issue :-) It generally works with Symfony 2.8 (that's probably why the test didn't fail), but the colon notation (`Bundle:folder:view.html.twig`) does not work anymore... Does anything speak against using the [`TemplateNameParser`](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Templating/TemplateNameParser.php) here?